lms/data_tasks/report/create_from_scratch/01_functions/01_date_functions.sql

```diff
@@ -36,6 +44,7 @@ AS $$
         WHEN resolution = 'all_time' THEN INTERVAL '200 years'
         WHEN resolution = 'year' THEN INTERVAL '1 year'
         WHEN resolution = 'academic_year' THEN INTERVAL '1 year'
+        WHEN resolution = 'trailing_year' THEN INTERVAL '1 year'
         WHEN resolution = 'semester' THEN INTERVAL '6 months'
         WHEN resolution = 'quarter' THEN INTERVAL '3 months'
         WHEN resolution = 'month' THEN INTERVAL '1 month'
```

Comment:
A bit later on there's a function to present the date. I don't really know how to present the trailing year, so I've left it as presenting it as a date. This will show the start date of the year. I'm not sure if there's a better way?

Something fancy like "1 year ago"? Seems like this would be long and difficult?